utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -1456,7 +1456,13 @@ int main(int argc, char **argv)
 
         if (testType == 0)
         {
-            cout << "\n\nGPU Backend Wall Time: " << wallTime <<" ms/batch"<< endl;
+            if(!qaFlag)
```

Comment:
You can restructure to something like below?
Also add a comment explaining the condition check
```
if (testType == 0)
{
    cout <<"\n\n";
    if(!qaFlag && noOfIterations > 1)
        cout <<"Execution Timings for Iteration "<< iterCount + 1 <<":"<<endl;
    cout << "GPU Backend Wall Time: " << wallTime <<" ms/batch";
```

Reply:
Done